src/codeu/chat/server/Server.java

```diff
@@ -56,7 +55,7 @@
   private Uuid lastSeen = Uuids.NULL;
 
   private final Database database;
-  private final UserSchema userSchema;
+  private UserTable userTable = null;
```

Comment:
`= null` is not required for member fields in java. Newly constructed objects in java have all uninitialized fields initialized to `null`.